utility.py

```
@@ -1,24 +1,31 @@
+config_contents = config.read(CONFIG)
 
 # Backup config
 config_template = configparser.ConfigParser()
-config_template_contents = config_template.read(config_template_name)
+config_template_contents = config_template.read(CONFIG_TEMPLATE)
 
 # Check for empty/missing config
 if not config_contents:
-    # TODO: Copy template config here instead of raising an error
-    raise RuntimeError(f"Unable to parse {config_name} - does it exist?")
+    warnings.warn(f"Unable to parse {CONFIG} - does it exist? Falling back to default config", RuntimeWarning)
```

Ruff (0.14.0)

No explicit `stacklevel` keyword argument found

Set `stacklevel=2`

(B028)